infrastructure/restapi/src/testFixtures/java/tech/pegasys/teku/infrastructure/restapi/StubRestApiRequest.java

```diff
@@ -57,6 +57,7 @@ public class StubRestApiRequest implements RestApiRequest {
   private final Map<String, String> optionalQueryParameters = new HashMap<>();
   private final Map<String, List<String>> listQueryParameters = new HashMap<>();
   private final Map<Integer, String> contentTypeMap = new HashMap<>();
+  private final Map<String, String> contextHeaders = new HashMap<>();
```

Comment:
I think a better naming would be `requestHeaders` to represent the headers we are reading from the incoming request, and `responseHeaders` to represent the headers we are adding to the response.